config/default_records/tube_purposes/005_cardinal_tube_purposes.yml

```diff
@@ -0,0 +1,7 @@
+# Note: Tubes in the cardinal pipeline are under control of Limber. These are
+# the initial tubes that enter via manifest
+---
+LCA Blood Tube:
+  type: Tube::Purpose
+  target_type: SampleTube
+  stock_plate: false
```

Comment:
Remind me how these are run?

Reply:
There is a task:
```bash
bundle exec rake record_loader:all
```
That gets triggered automatically on deployment. (After migrations have run)